volume-group-operator/controllers/persistentvolumeclaim/persistentvolumeclaim_controller.go

```diff
@@ -32,14 +32,14 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/builder"
 	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/predicate"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
 type PersistentVolumeClaimWatcher struct {
 	Client            client.Client
 	Scheme            *runtime.Scheme
 	Log               logr.Logger
+	DriverConfig      *config.DriverConfig
```

Comment:
while you are adding the DriverConfig, plz add some check that we are the PVC provisioners
something like
```
if r.DriverConfig.DriverName != sc.Provisioner{
   return ctrl.Result{}, nil
}
(need to get storageclass)
```

Reply:
done [in](https://github.com/IBM/ibm-block-csi-operator/pull/302/commits/6848de34150f17e3cd1fbdadac7d8143b5776fe0)